spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/model/instance/AsyncParameterizedServiceInstanceRequest.java

```diff
@@ -36,6 +36,7 @@
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public abstract class AsyncParameterizedServiceInstanceRequest extends AsyncServiceBrokerRequest {
 
+	@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Hadn't realised we'd need this, but the `@JsonInclude(JsonInclude.Include.NON_NULL)` of `CreateServiceInstanceBindingRequest` overrides this, and so without the annotation on the field we start seeing empty maps in the serialised JSON.